### ISSUE TITLE
Enemys

### ISSUE THREAD

**Issue by oflatt** [oflatt/space-orbs#14]
Currently, because it is small, you are able to kill your teammates. In the future this should not be the case. The deathmatch mode would be similar to right now, but once you are dead you would have to wait until the round ends. The round would end when one team has all members dead, and I can't decide if I want to include friendly fire. It would be nicer if the server and client keep track of the different teams, which requires changes of structure.

**Comment by AlexKnauth** [comment]
Um, it isn't important for players to check whether it's a teammate before shooting?

**Comment by oflatt** [comment]
Yes, and because there is no collision between players I think friendly fire would a fine addition to the future deathmatch gamemode.

**Comment by oflatt** [comment]
Once I have time I will make the orbs struct include player, enemys, and teammates.
